service/raft/group0_state_machine.hh

```diff
@@ -37,8 +37,13 @@ struct topology_change {
     std::vector<canonical_mutation> mutations;
 };
 
+// This command is used to write data to tables over raft.
```

Comment:
Please provide more explanation in the comment as described here:
https://github.com/scylladb/scylladb/pull/13962#discussion_r1200645161
specifically, this part:
> write a comment at struct write_mutations (in group0_state_machine.hh) explaining that this command is used to write data to tables **other than topology or schema tables and it doesn't update any in-memory data structures**

Reply:
Fixed.